spec/models/searchgov_url_spec.rb

```diff
@@ -125,6 +130,17 @@
         fetch
       end
 
+      context 'when the record is enqueued for reindex' do
+        let(:searchgov_url) do
+          SearchgovUrl.create!(valid_attributes.merge(enqueued_for_reindex: true))
+        end
+
+        it('sets enqueued_for_reindex to false') do
```

Comment:
@peggles2 , we don't include parentheses in `it` blocks for stylistic consistency:
```ruby
it 'sets enqueued_for_reindex to false'  do
```

Reply:
doup! good catch! :) 👍